README.md

```diff
@@ -49,9 +49,10 @@ $ go get github.com/gofrs/uuid
 
 ## Requirements
 
-This package is regularly tested against Go 1.3+, due to issues with the testing
-libraries on Go 1.2 or less. This package may work perfectly fine with Go 1.2,
-but it is not tested and support for this version is not actively maintained.
+This package is regularly tested against Go 1.7+, due to older versions not
+supporting subtests. This package may work perfectly fine with Go 1.2+, but it
+is not tested against and support for these older versions is not actively
```

Comment:
> but it is not tested against and support for these [...]

This phrasing sounds a little odd to me. It may be because I'm not a native English speaker, but I had to read it a couple of times to understand the meaning.

Reply:
@acln0 patch applied -- how's it look now?